samples/sensor/apds9960/README.rst

```diff
@@ -1,52 +1,38 @@
-.. _apds9960-sample:
+.. _apds9960:
 
-RGB and Gesture Sensor
-######################
+APDS9960 RGB, ambient, Gesture Sensor
+#####################################
 
 Overview
 ********
 
-This sample utilizes APDS-9960 Sensor and reads RGB values from the sensor
-then displays the color through the APA102C LED.
+A sensor application that demonstrates how to use the APDS9960 to get ambient light,
+RGB and proximity data. This sample only checks sensor in polling mode(without trigger)
 
+Building and Running
+********************
 
-.. note::
-   This sample does not use the Zephyr sensor APIs
-
-Wiring
-******
-
-The SparkFun RGB and Gesture Sensor was being used:
-
-- https://www.sparkfun.com/products/12787
-- https://www.adafruit.com/product/2343
-
-For APA102C, on the sensor subsystem (ARC) side of Arduino 101:
-
-1. GPIO_SS_2 is on AD0 (for APA102C data)
-2. GPIO_SS_3 is on AD1 (for APA102C clock)
-
-The GPIO driver is being used for bit-banging to control the APA102C LED.
-
-The APA102/C requires 5V data and clock signals, so logic level shifter
-(preferred) or pull-up resistors are needed.  Make sure the pins are 5V
-tolerant if using pull-up resistors.
+samples provide output to the console. It requires an apds9960 sensor and 
```

Comment:
This sample application uses an APDS9960 sensor connected to an Arduino 101 board
    (for example as shown in this `Sparkfun APDS9960 tutorial<https://www.sparkfun.com/products/12787>`__.
    There are two processor cores (x86 and ARC) on the Arduino 101.  You'll need to flash
    both stub code on the x86 core and this sample's code on the ARC core, as explained below.